analyses/Data_transformation.R

```diff
@@ -0,0 +1,119 @@
+target_feature_value <- params$added_feature
+target_feature_shap <- params$added_feature_shap
+nbhd <- ccao::nbhd_shp
+
+# Create a individual card level dataset
+card_individual <- shap_new %>%
+  select(
+    meta_pin, meta_card_num, pred_card_shap_baseline_fmv,
+    {{ target_feature_value }}
+  ) %>%
```

Comment:
Renames the value to append _shap, allowing the feature value to be merged without creating .x, .y.